web/src/main/resources/static/index.html

```diff
@@ -103,7 +103,8 @@ <h3>Mapping Tools</h3>
             <div class="col-md-12">
               <div class="card bg-light mb-3">
                 <div class="card-body">
-                  <div id='oncotree-mapping-tool'/>
+                  <div id='oncotree-mapping-tool'>
+                  </div>
```

Comment:
Ha! @ao508 requested that I change <div></div> to <div />.  Now @sheridancbio wants to undo that. :)